app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1726,12 +1721,24 @@ BrowserWorld::DrawWorld(device::Eye aEye) {
     m.drawList->Draw(*camera);
   }
 
-  // Draw hand models
+  // Draw hand mesh if active
```

Comment:
That's what we do. Draw() is here only for the generic hand mesh rendering code (the Skinned strategy) that don't use VRB at all.

On the Spheres strategy, Draw() is empty precisely because we integrate into the scene graph (the spheres are added to the root node of ControllerContainer).

The Geometry strategy (for MSFT extension), will also use the scene graph.

But we still need a Draw() here for now, to support the Skinned. I have plans to move Skinned to the scene graph as well, once I land the changes to VRB that I have been working on.

Reply:
Right, I already mentioned that we do that for the spheres case. I was wondering whether we could make the exception more exceptional :). I mean we're adding `Draw()` to the strategy interface but it turns out that it is only temporarily used in one case. That's why I was wondering why don't just avoid defining `Draw()` and handle the exceptional case separatedely?